### DISCUSSION TITLE
conda : The term 'conda' is not recognized as the name of a cmdlet,

### DISCUSSION THREAD

**Discussion by DiekiRianMustapa** [teamalgoritma/community#194] [Q&A] [answered]
![Masalah](https://user-images.githubusercontent.com/107328516/178149142-b3083f5c-09e4-4ba2-9299-6575c9d3d908.png)

Apakah bisa dibantu ? setelah dirunning terminal muncul informasi seperti ini, padahal secara skeleton IPYNB nya sudah jalan semua

**Comment by triarm** [comment, marked as answer]
Hi @DiekiRianMustapa ,
error tersebut terjadi karena terminal yang digunakan adalah PowerShell bukan Command Prompt. Untuk mengubah terminal tersebut menjadi Command Prompt, Anda dapat melihat solusi pada [diskusi ini](https://github.com/teamalgoritma/community/discussions/180)

Demikian yang dapat kami sampaikan. Semoga menjawab.
